zmon-controller-ui/js/controllers/EntityCtrl.js

```diff
@@ -17,6 +17,53 @@ angular.module('zmon2App').controller('EntityCtrl', ['$scope', '$window', '$rout
             var timeIntervalSinceLastUpdate = MainAlertService.millisecondsApart(epochPastTs, MainAlertService.getLastUpdate());
             return timespanFilter(timeIntervalSinceLastUpdate);
         };
+
+        $scope.numberNiceFormat = function(v) {
+            var ndx = Math.floor( Math.log(v) / Math.log(1000) );
+            var suffix = ["", "K", "M", "G", "T", "P"];
+            return  (v / Math.pow(1000, ndx)).toFixed(1) + suffix[ndx];
+        };
+
+        $scope.getDisplayValue = function(r) {
+            if (!r.result) return (r.status == 'inactive' ? 'OK' : '?');
+            var v = null;
+            if (Object.keys(r.result.captures).length == 1) {
+                v = r.result.captures[Object.keys(r.result.captures)[0]];
+                if (typeof v == "number") {
+                    if (v < 0) {
+                        return (v * 100).toFixed(0) + "%";
```

Comment:
This is obviously wrong, I wanted to write 0 < x < 1 . sounded like a meaningful "guess"